README.md

```diff
@@ -119,7 +119,7 @@ repository
 Before committing any documents to GitHub, please ensure any fields containing sensitive data are
 anonymised.
 
-**Complete the following steps to add another test case**:
+**Complete the following steps to add another test case, NEEDS UPDATING IF WE CONTINUE TO ADD - CURRENTLY OUT OF DATE**:
```

Comment:
TODO

Reply:
does that mean TODO now or do we need a ticket?

Reply:
I would prefer a tech debt ticket for the updates to the README

Reply:
I'll create a tech debt ticket then and link it here - happy with that approach @cgregoryCH ?

Reply:
Works for me